src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -856,92 +861,37 @@ public void assertRootNodeOrBuiltNode(UUID studyUuid, UUID nodeUuid) {
         }
     }
 
-    private LoadFlowParameters getLoadFlowParameters(StudyEntity studyEntity) {
-        return LoadFlowService.fromEntity(studyEntity.getLoadFlowParameters());
-    }
-
     public LoadFlowParameters getLoadFlowParameters(UUID studyUuid) {
-        return studyRepository.findById(studyUuid)
-                .map(this::getLoadFlowParameters)
-                .orElse(null);
-    }
-
-    public LoadFlowParametersInfos getLoadFlowParametersInfos(StudyEntity study) {
-        LoadFlowParameters commonParameters = getLoadFlowParameters(study);
-        List<LoadFlowSpecificParameterInfos> specificParameters = getSpecificLoadFlowParameters(study, ComputationUsingLoadFlow.LOAD_FLOW);
-        return LoadFlowParametersInfos.builder()
-                .commonParameters(commonParameters)
-                .specificParameters(specificParameters.stream().collect(Collectors.toMap(LoadFlowSpecificParameterInfos::getName, LoadFlowSpecificParameterInfos::getValue)))
-                .build();
+        LoadFlowParametersValues lfParameters = getLoadFlowParametersValues(studyUuid);
+        return lfParameters.getCommonParameters();
     }
 
     public LoadFlowParametersValues getLoadFlowParametersValues(UUID studyUuid) {
-        StudyEntity study = studyRepository.findById(studyUuid).orElseThrow(() -> new StudyException(STUDY_NOT_FOUND));
-        LoadFlowParameters commonParameters = getLoadFlowParameters(study);
-        List<LoadFlowSpecificParameterInfos> specificParameters = getAllSpecificLoadFlowParameters(study);
-        Map<String, Map<String, Object>> specificParametersPerProvider = specificParameters.stream()
-            .collect(Collectors.groupingBy(LoadFlowSpecificParameterInfos::getProvider,
-                Collectors.toMap(LoadFlowSpecificParameterInfos::getName, LoadFlowSpecificParameterInfos::getValue)));
-        return LoadFlowParametersValues.builder()
-                .commonParameters(commonParameters)
-                .specificParametersPerProvider(specificParametersPerProvider)
+        StudyEntity studyEntity = studyRepository.findById(studyUuid).orElseThrow(() -> new StudyException(STUDY_NOT_FOUND));
+        UUID loadFlowParametersUuid = studyEntity.getLoadFlowParametersUuid();
+        if (loadFlowParametersUuid == null) {
+            return LoadFlowParametersValues.builder()
+                .commonParameters(LoadFlowParameters.load())
+                .specificParametersPerProvider(Map.of())
                 .build();
+        }
```

Comment:
For me it's the job of the loadflow-server to give whatever it wants when we ask for parameters without giving an UUID. We shouldn't have half the logic in the study-server and the other half in the loadflow server, it kinda defeat the whole point of handling them in the loadflow-server no ?

Reply:
I added a new endpoint in the loadflow-server that allows creating default parameters that are now associated with the study when it is created